src/Engine/ProtoScript/Runners/LiveRunner.cs

```diff
@@ -18,7 +18,7 @@ namespace ProtoScript.Runners
     /// <summary>
     /// A subtree represents a node in graph. It contains a list of AST node.
     /// </summary>
-    public struct Subtree
+    public class Subtree
```

Comment:
@mjkkirschner is it okay to break this considering we don't expect any external devs to be using APIs from ProtoScript/ProtoCore?